apps/start-execution-worker/src/start_execution_worker.py

```diff
@@ -8,24 +9,35 @@
 
 STEP_FUNCTION = boto3.client('stepfunctions')
 
+batch_params_file = Path(__file__).parent / 'batch_params_by_job_type.json'
+if batch_params_file.exists():
+    BATCH_PARAMS_BY_JOB_TYPE = json.loads(batch_params_file.read_text())
+else:
+    # Allows mocking with unittest.mock.patch
+    BATCH_PARAMS_BY_JOB_TYPE = {}
```

Comment:
It may not be worth the effort right now, but I'm pretty sure you can patch `Path.exists`, `Path.read_text`, and `json.loads`.

Reply:
Here are some references if you want them:
* [How to patch exists](https://stackoverflow.com/questions/48864027/how-do-i-patch-the-pathlib-path-exists-method)
* [How to patch json.loads](https://stackoverflow.com/questions/51138834/how-to-use-mock-open-with-json-load)